qiskit/visualization/matplotlib.py

```diff
@@ -548,6 +555,7 @@ def _draw_regs(self):
                 len_longest_label = len(label)
 
             pos = -ii
+            label = _fix_double_script(label)
```

Comment:
Should this be moved above like 554? That line is doing a check on the length of the label, and this call can change the length of the label

Reply:
Yep, I also spotted this while testing.

I had assumed that this was intentional.

There was some other weird behaviour with spaces where they are removed from the label, but are considered when calculating the spacing:

![image](https://user-images.githubusercontent.com/58116376/74428414-dbc9b100-4e50-11ea-9bd9-472359066318.png)

Reply:
I would have assumed it wasn't intentional, or at least it isn't standard across the drawers as the text drawer does write labels with the spaces in them. 

That definitely doesn't seem like sensible behaviour to me so I think this should be opened as a separate issue.